CairoFCCOS/FCCOSEth002.xml

```diff
@@ -0,0 +1,218 @@
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="81" to="120"/>
+                                <title type="complete" ref="LIT1146Argano#Wednesday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="121" to="159"/>
+                                <title type="complete" ref="LIT1146Argano#Thursday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.5">
+                                <locus from="160" to="192"/>
+                                <title type="complete" ref="LIT1146Argano#Friday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.6">
+                                <locus from="193" to="210"/>
+                                <title type="complete" ref="LIT1146Argano#Saturday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.7">
+                                <locus from="210" to="226"/>
+                                <title type="complete" ref="LIT1146Argano#Sunday"/>
+                            </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="227" to="230"/>
+                            <title type="incomplete" ref="LIT2842RepCh125" cert="low"/>
+                            <textLang mainLang="gez"/> 
+                            <note>The text, incomplete at the beginning and end, is a malkǝʾ-hymn to Christ, apparently
+                                the <ref type="work" corresp="LIT2842RepCh125"/>. The catalogue description does not allow
+                            to ascertain whether this text is part of the original production, a later textual addition, or a material addition
+                            originating from another manuscript.</note>
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="page" quantity="230">230</measure>
+                                    <dimensions type="outer" unit="cm">
+                                        <height>20.5</height>
+                                        <width>18</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">The binding is broken.</condition>
```

Comment:
also repeated under binding below? needed twice?

Reply:
I agree it is redundant. The reason I encoded it in this way is that this one sentence is the only information we have on the actual presence of a binding, and it equally applies to the condition. What would you propose?

Reply:
Am not sure, lets leave it